smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -38,7 +41,15 @@ open class HttpProtocolServiceClient(
                     }
                     prop.renderInitialization(writer, "config")
                 }
-
+                // Render auth schemes
+                writer.write("var modeledAuthSchemes: [ClientRuntime.AuthScheme] = Array()")
+                if (ServiceIndex(ctx.model).getEffectiveAuthSchemes(ctx.service).contains(SigV4Trait.ID)) {
+                    writer.write("modeledAuthSchemes.append(SigV4AuthScheme())")
+                }
+                if (ServiceIndex(ctx.model).getEffectiveAuthSchemes(ctx.service).contains(SigV4ATrait.ID)) {
+                    writer.write("modeledAuthSchemes.append(SigV4AAuthScheme())")
+                }
+                writer.write("config.authSchemes = config.authSchemes ?? modeledAuthSchemes")
```

Comment:
Add to client config initializer, the code that either takes user's custom auth schemes from the config or puts in default auth schemes supported by both SDK and service. Right now it's only SigV4 and SigV4A.